src/event-helper-listen.js

```diff
@@ -41,11 +43,34 @@ export function internalListenImplementation(element, eventType, listener,
       throw e;
     }
   };
+
+  // Test whether browser supports the passive option or not
+  let passiveSupported = false;
```

Comment:
Cache this value in a global and only run the feature detection code once per window.